image/build-container-image.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script uses docker buildx to build Windows container images
+# on linux to run csi-proxy.exe inside HostProcess containers.
+# Building Windows images on Linux requests setting --output=type=registry on the build commands.
+
+set -o nounset
+set -ex
+
+: "${VERSION:?VERISON not set}"
+: "${REGISTRY:?REGISTRY not set}"
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+if [ $(docker buildx ls | grep -c img-builder) == 0 ]; then
+    docker buildx create --name img-builder 
+fi
+docker buildx use img-builder
```

Comment:
might consider cleaning up the image builder:

`trap 'docker buildx rm img-builder' EXIT`